### DISCUSSION TITLE
Change: "Work in progress, please don't use as reference"?

### DISCUSSION THREAD

**Discussion by gnizla** [SChAth/ScAthiop#1855] [Ideas]
Dear All,

Just found myself reading a contribution on BM by @thea-m  which contains the usual refrain "_Work in progress, please don't use as reference_". Given that the record is in fact very detailed I could not help myself from wondering whether it might not be better/more fair, to slightly alter the phrasing for citation so "_Work in Progress_" taking away the "_please don't use as reference part_"? 

It is good to emphasize that this is work in progress, but then when will it not be in progress? This is a very good record already. With this wording, the risk is that it discourages people using BM from actually citing it and this goes against the app's and the encoder's interests.

Just a thought,

J

**Comment by PietroLiuzzo** [comment]
I agreed. It should be already possibile and up to the editor to write "completed" in a change element to make that mark change from work in progress. There is also an under review stage and more can be added.
